drivers/ethernet/eth_smsc911x_priv.h

```diff
@@ -0,0 +1,214 @@
+/* mbed Microcontroller Library
```

Comment:
Merge this with next patch: driver code should come all at once.

Reply:
Done.